src/components/enketo-iframe.vue

```diff
@@ -102,11 +107,26 @@ function handleIframeMessage(event) {
     let eventData;
     try { eventData = JSON.parse(event.data); } catch {}
 
-    if (eventData?.enketoEvent === 'submissionsuccess' && redirectUrl.value) {
-      router.push((new URL(redirectUrl.value)).pathname);
+    if (eventData?.enketoEvent === 'submissionsuccess') {
+      if (props.actionType === 'edit') {
+        // for edit we always redirect to Submission details page
+        router.push(submissionPath(form.projectId, form.xmlFormId, props.instanceId));
```

Comment:
Do we have a test of this case already (redirect to submission detail page after Enketo edit)?

Reply:
We have one for WebForms, but not for Enketo. 

Added one